src/sakit/unused_level_select.c

```diff
@@ -14,8 +14,6 @@
 
 #include "constants/songs.h"
 
-const char ALIGNED(4) gUnknown_080D5128[8] = "STAGE";
```

Comment:
Where is this now?

Reply:
In the function call! ^^

Reply:
Wow, very good! I'm sure there are other places where this could be done